Kconfig

```diff
@@ -24,6 +24,30 @@ config DOM_CFG_BOARD_EXT
 	    rpi_5_xenvm.c
 	    rpi_5_domu.c
 
+config DOM_STORAGE_FATFS_ENABLE
+	bool "Enable FATFS disk storage for reading domains kernels/dtb images"
+	default n
+	select DISK_ACCESS
+	select FILE_SYSTEM
+	select FAT_FILESYSTEM_ELM
+	select XRUN
```

Comment:
It is strange to enable whole `XRUN` functionality only because we need store access functions. Can we make them generic?

Reply:
Yes and no. Yes in general. No - not as part of this submission/feature.